### ISSUE TITLE
PHP Biscuit doesn't handle brackets on new lines well

### ISSUE THREAD

**Issue by mwnciau** [code-biscuits/assorted-biscuits#7]
e.g.

    if ( $success )
    {
        $newDonor = $_POST['SERIALNUMBER'];
    }

Shows the biscuit:

![image](https://user-images.githubusercontent.com/2857218/130423041-b7094404-4f60-4954-bc73-f83a3db3974e.png)

Presumably an issue in other languages, too, where the coding style is to place brackets on a new line.

**Comment by mwnciau** [comment]
On a second look, this might be intended behaviour, but nonetheless I find it confusing. This only happens after the if block in an if-else statement.

![image](https://user-images.githubusercontent.com/2857218/130484198-310ab2bf-93bb-4896-8847-2f3322f1e9c0.png)

It seems more intuitive to me that the // if ( $condition ) should come at the end of the if block, not at the end of the else block.

**Comment by cmgriffing** [comment]
Yeah, I agree. The output will need some fine tuning. That is my mistake. My testing dataset just doesn't have that formatting in it. I will work on that this weekend probably.

**Comment by cmgriffing** [comment]
I did some fiddling.

Here is the result:
![Screen Shot 2021-08-29 at 5 23 22 PM](https://user-images.githubusercontent.com/1195435/131270438-730f3497-eec8-4446-aaa6-33918ea8ee09.png)

It isn't perfect, but I think it improves on the else with brackets on the same line as well. I'm thinking the if being shown on the very last part of an else is intended at this point. We just annotate the middle bracket now too.

If you want to toy around with it more, I have a working branch going:
https://github.com/code-biscuits/assorted-biscuits/tree/language-specific-filters

If you think its good enough, I can get a release out after some cleanup.
